### ISSUE TITLE
byteBuffer.rewind():  no such method

### ISSUE THREAD

**Issue by dacase** [mjschnie/ffx#6]
I tried this command:

  ffxc xray.scaleBulk --mtz lys_tricl_4_refmac12.pdb lys_tricl_4_refmac12.mtz

If I leave out the "--mtz" flag, it works fine; but with that flag I get this error:

 Writing MTZ HKL file: "lys_tricl_4_refmac12_ffx.mtz"

Exception in thread "main" java.lang.NoSuchMethodError: java.nio.ByteBuffer.rewind()Ljava/nio/ByteBuffer;
    at ffx.xray.parsers.MTZWriter.write(MTZWriter.java:301)

Googling the error message didn't help (at least for me!).  I'm on MacOS with jdk1.8.0_171, built from today's github using maven 3.2.5.    If I use ffxc from the binary build of 05-25-2018, it works fine, so this is low priority; I just like to feel that I can potentially modify the code to do new things.

**Comment by jacob-litman** [comment]
Hello,

I'm afraid I'm unable to replicate with our standard test case (ffx/examples/alamet.[pdb|cif|properties]). Would it be possible to attach these files? My guess is there's something about how the Refmac .mtz is formatted that is giving FFX issues.

Alternately: try these steps:
Navigate to your FFX installation
mvn clean
mvn deploy
Test your command

mvn clean
mv ~/.m2 ~/bak_m2
mvn deploy
Test your command

The "mvn clean" command should remove any artifacts from prior builds, and ~/.m2 is the default Maven repository, with all the dependency packages. Renaming/removing the .m2 directory forces Maven to build FFX with all freshly obtained dependency packages.

Thank you,
Jacob Litman

**Comment by dacase** [comment]
Sorry to be slow here--I got busy with lots of other stuff.  The commands:

mvn clean
mvn deploy
Test your command

did the trick!  Earlier, I had just built with "mvn" (not "mvn deploy").  Not sure if that is key to this or not.  Anyway, thanks for your help.

**Comment by jacob-litman** [comment]
Hello Prof. Case,

First, glad to know it worked for you.

Second, "mvn" should act the same as "mvn deploy"; I just tend to do it longhand because I joined the lab before we configured "deploy" to be the default Maven lifecycle command.

Since it seems the case is resolved, I'll go ahead and close the issue, and consider adding a quick troubleshooting guide to the repository.

Thank you,
Jacob Litman
